modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * @file
+ * The iNope Policy Review Module.
+ */
+
+/**
+ * Implements hook_menu().
+ */
+function inope_policy_review_menu() {
+  $items = array();
+
+  $items['node/%/review'] = array(
+    'title callback' => 'inope_policy_review_tab_title_callback',
+    'title arguments' => array(1),
+    'description' => 'Review of policy nodes.',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('inope_policy_review_form'),
+    'access arguments' => array('view published content'),
+    'type' => MENU_LOCAL_TASK,
+  );
+
+  return $items;
+
+}
+
+/**
+ * The iNope Policy Review Page Title Callback.
+ */
+function inope_policy_review_tab_title_callback($node_nid) {
+  $node = node_load($node_nid);
+  $node_title = $node->title;
+  $title = t('Review of @node_title', array('@node_title' => $node_title));
+  return $title;
+}
+
+/**
+ * The iNope Policy Review Form.
+ */
+function inope_policy_review_form($form, &$form_state) {
+
+  $form['inope_policy_review_approved_checkbox'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('I have reviewed this policy and approved it for use'),
+    '#default_value' => TRUE,
+    '#states' => array(
+      'unchecked' => array(
+        ':input[name="inope_policy_review_changes_checkbox"]' => array('checked' => TRUE),
+      ),
+      'checked' => array(
+        ':input[name="inope_policy_review_changes_checkbox"]' => array('checked' => FALSE),
+      ),
+    ),
+  );
+
+  $form['inope_policy_review_changes_checkbox'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('I have reviewed this policy and would like to see changes'),
+    '#default_value' => 0,
+    '#states' => array(
+      'checked' => array(
+        ':input[name="inope_policy_review_approved_checkbox"]' => array('checked' => FALSE),
+      ),
+      'unchecked' => array(
+        ':input[name="inope_policy_review_approved_checkbox"]' => array('checked' => TRUE),
+      ),
+    ),
+  );
+
+  $form['inope_policy_review_changes_requested_textarea'] = array(
+    '#type' => 'textarea',
+    '#title' => t('Changes Requested:'),
+    '#default_value' => '',
+    '#states' => array(
+      'invisible' => array(
+        ':input[name="inope_policy_review_changes_checkbox"]' => array('checked' => FALSE),
+      ),
+      'required' => array(
+        ':input[name="inope_policy_review_changes_checkbox"]' => array('checked' => TRUE),
+      ),
+    ),
+  );
+
+  $form['inope_policy_review_submit'] = array(
+    '#type' => 'submit',
+    '#value' => t('Submit'),
+  );
+
+  return $form;
+}
+
+/**
+ * The iNope Policy Review Form submit handler.
+ */
+function inope_policy_review_form_submit($form, &$form_state) {
+  $user = $GLOBALS['user'];
```

Comment:
You actually want to be pretty careful with a variable called user.  I also think you don't want to get the user object in this way.

Look here: https://api.drupal.org/api/drupal/modules!user!user.module/function/user_access/7

See what they do, use this approach. Head to get user into $account as quickly as possible to protect yourself. $account is more abstract and a good way to avoid problems when dealing with all "user" objects.